pkg/network/vmispec/interface.go

```diff
@@ -171,3 +171,14 @@ func FilterInterfacesByNetworks(interfaces []v1.Interface, networks []v1.Network
 	}
 	return ifaces
 }
+
+func BindingPluginNetworkWithDeviceInfoExist(ifaces []v1.Interface, bindingPlugins map[string]v1.InterfaceBindingPlugin) bool {
```

Comment:
The `BindingPluginNetwork` confusing it should be `NetworkBindingPlugin` as  the feature name IMO.

Reply:
Now I understand what it means, maybe mentioning interface fit better a because the function receives interfaces slice.
Although, we already know the API suffer from the network/interface naming semantics, so maybe network can still work.

Still a nit, not a must.